bot/handlers/weather.py

```diff
@@ -60,6 +60,9 @@ def weather(config, bot, update, args):
     uvi = owm.uvindex_around_coords(lat, lon).get_value()
     wind = now_weather.get_wind()
 
+    wind_speed = wind.get("speed") if wind.get("speed") else 0
```

Comment:
Does `wind.get("speed",DEFAULT_VALUE)` construction not works in this case?

Reply:
Fuck, I need more coffee, thanks